Client Config nav reorg (#382)
uyuni-project/uyuni-docs: modules/client-configuration/pages/system-locking.adoc

====

For more information about blackout mode in Salt, see https://docs.saltstack.com/en/latest/topics/blackout/index.html.

== Package Locks

[NOTE]
====
Package locks can only be used on traditional clients that use the Zypper package manager.
The feature is not currently supported on {rhel} or Salt clients.
====

Package locks are used to prevent unauthorized installation or upgrades to software packages on traditional clients.

> When a package has been locked, it will show a padlock icon, indicating that it can not be installed.

Reviewer: ```suggestion
When a package has been locked, it will show a padlock icon, indicating that it cannot be installed.
```

Any attempt to install a locked package will be reported as an error in the event log.

> Locked packages can not be installed, upgraded, or removed, either through the {productname} {webui}, or directly on the client machine using a package manager.

Reviewer: ```suggestion
Locked packages cannot be installed, upgraded, or removed, either through the {productname} {webui}, or directly on the client machine using a package manager.
```

Locked packages will also indirectly lock any dependent packages.

.Procedure: Using Package Locks
. On the client machine, install the [package]``zypp-plugin-spacewalk`` package as [systemitem]``root``:
+
----
zypper in zypp-plugin-spacewalk
----

. Navigate to the menu:Software[Packages > Lock] tab on the managed system to see a list of all available packages.
. Select the packages to lock, and click btn:[Request Lock].
You can also choose to enter a date and time for the lock to activate.
Leave the date and time blank if you want the lock to activate as soon as possible.
Note that the lock might not activate immediately.
. To remove a package lock, select the packages to unlock and click btn:[Request Unlock].
Leave the date and time blank if you want the lock to deactivate as soon as possible.

> Note that the lock might not deactivate immediately.

Reviewer: ```suggestion
The lock might not deactivate immediately.
```
